jsapp/js/components/processing/translations/translationsTab.component.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import clonedeep from 'lodash.clonedeep';
 import {formatTime} from 'js/utils';
-import singleProcessingStore from 'js/components/processing/singleProcessingStore';
-import TransxAutomaticButton from 'js/components/processing/transxAutomaticButton';
+import singleProcessingStore from '../singleProcessingStore';
```

Comment:
My only question is why these absolute paths for `singleProcessingStore` and `TransxAutomaticButton` are getting changed to relative paths. It's slightly fewer characters, but if you, say, moved this file to a different folder again, you'd have to edit these imports - unnecessary if you leave it as an absolute path.

Reply:
Good point :)